### DISCUSSION TITLE
i got  TimeField with label 开始生效时间  not found,Why?how do i ues timeField api

### DISCUSSION THREAD

**Discussion by labulakaupppppp** [getgauge/taiko#2710] [General]
Pic1
<img width="966" alt="image" src="https://github.com/getgauge/taiko/assets/26108549/35cf274b-5fa4-47a8-8899-4104f8e4c58f">
Pic2:
<img width="1008" alt="image" src="https://github.com/getgauge/taiko/assets/26108549/49821996-f152-4bfd-82a0-6daf228238a9">

 the page just like this picture, when i click the input type=text which text us "开始生效时间",the timefield will open
My question 1: 
code: await click('开始生效时间');  but the timeField is not open

My question 2: 
i open the timeField my hand and input code like:
await write (timeStr,into(timeField('开始生效时间')))
and i got an error 【Error: TimeField with label 开始生效时间  not found, run `.trace` for more info】
so i want to know ,how  can i input the time into the timeField?
Thank you very much!

**Comment by fhirter** [comment]
I was able to resolve this issue by using a label with `for` to connect it to the input field. See my comment on my similar issue: https://github.com/getgauge/taiko/issues/2740

This answers your second question. Regarding your first question I would avoid trying to use the date picker. I think its fair to rely on the browser to put a valid date into the field when using the date picker.
